web/app/Http/Controllers/AppController.php

```diff
@@ -183,4 +189,33 @@ public function postDevModeLaunch(Request $request) {
 
         return redirect('/app');
     }
+
+    public function gradePassback(Request $request){
+          $assignment = Assignment::where('resource_link_dbid', $request->session()->get('lti_resource_link_dbid'))
+         ->firstOrFail();
+            
+         set_time_limit(60);
+         ini_set('memory_limit', '2048M');
+         if(!ini_get('auto_detect_line_endings')){
+            ini_set('auto_detect_line_endings', '1');
+         }
+
+         $assignment_response = AssignmentResponse::where('assignment_id',$assignment->id)
+         ->get();
+         if($assignment_response->count() == 0){
+            return redirect('/app');
+         }
+        
+         $csv = Writer::createFromFileObject(new SplTempFileObject());
+    
+         $csv->insertOne(['timeStamp' , 'userResultID', 'score', 'userName', 'userEmail']);
```

Comment:
Let's make two changes here:
* Remove the 'userResultID' header and column.  (Result IDs are database IDs and aren't meaningful to the end user.)
* Rename the 'timeStamp' header to indicate the event that this is a timestamp for.  'gradeReported' maybe?